materials/database-control-sql.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: page
+element: notes 
+title: Database Control
+language: SQL
+---
+
+> Open Excel spreadsheet. Fill in a couple of columns.
+
+> Open SQLite Manager, if it is not already open.
+
+### Database Control
+
+* There are many ways to add and modify data in a database management system.
+    * Enter the data into another program (e.g., a spreadsheet) and import it to SQLite.
+    * Get the data from somewhere and import it. 
+        * *Though, data isn't always ready to import into a database*
+    * Enter the data directly.
+
+### Enter the data into another program
+
+* Quality Assurance in Excel
+    * Select column.
+    * On the `Data` menu tab, select `Data Validation`.
+    * `Settings` tab
+        * `Allow:` selects type of data.
+        * `Data:` selects condition of value inputs
+            * `Value:`
+            * `Minimum:`
+            * `Maximum:`
+    * `Input Message` tab
+        * Show input message when cell is selected for entry.
+    * `Error Alert` tab
+        * `Style:` controls response to invalid input value
+            * `Stop`
+            * `Warning`
+            * `Information`  
+        
+> Do [Exercise 1 - Data entry validation in Excel]({{ site.baseurl }}/exercises/Qaqc-data-entry-validation-in-excel-SQL/). 
+
+### Import existing data 
+
+> Walk through [Exercise 2 - Importing Data]({{ site.baseurl }}/exercises/Database-control-importing-data-SQL), Tasks 1-10.
+
+> Do [Exercise 2 - Importing Data]({{ site.baseurl }}/exercises/Database-control-importing-data-SQL), Task 11.
+
+### Enter the data directly
+
```

Comment:
No worries. Just part of the process. I'll probably have to modify the exercise, too, if I understand you correctly.